app/uk/gov/hmrc/lisaapi/controllers/LisaActions.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.lisaapi.controllers
+
+import play.api.libs.json.Json.toJson
+import play.api.mvc.Results._
+import play.api.mvc._
+
+import scala.concurrent.Future
+
+trait LisaActions {
```

Comment:
@chriswaugh Please could you review this file?

Reply:
Yup I'm happy with it. Like we'd discussed it does have the downside of losing the auditing for these particular failures, but if they're not adding any values (which we don't believe they are) then it's the right thing to do.